RxBlocking/BlockingObservable+Operators.swift

```diff
@@ -10,14 +10,20 @@
     import RxSwift
 #endif
 
+public enum SequenceMaterializeResult<T> {
```

Comment:
Not sure on the naming here, 
Would expect `MaterializedSequenceResult<T>` perhaps

Also wondering if it could be nice to go with `elements` and `errors` similar to RxSwiftExt's operators that extend materialize?